### ISSUE TITLE
SEVERE: null when retrieving access token

### ISSUE THREAD

**Issue by duncte123** [thelinmichael/spotify-web-api-java#202]
Version info:
- Java: 11.0.6 (AdoptOpenJDK OpenJ9)
- Spotify API: 5.0.3
- OS windows 10

When turning on debug logging I can see that the tokens are being set correctly in my application yet this error message appears in my console
```
Apr 20, 2020 10:28:24 PM com.wrapper.spotify.SpotifyHttpManager execute
SEVERE: null
```

Code to reproduce:
```java
SpotifyApi spotifyApi = new SpotifyApi.Builder()
                .setClientId("Client_id")
                .setClientSecret("Client_secret")
                .build();

ClientCredentialsRequest request = spotifyApi.clientCredentials().build();
ClientCredentials clientCredentials = request.execute();

// Set access token for further "spotifyApi" object usage
spotifyApi.setAccessToken(clientCredentials.getAccessToken());

logger.debug("Successfully retrieved an access token! " + clientCredentials.getAccessToken());
logger.debug("The access token expires in " + clientCredentials.getExpiresIn() + " seconds");
```

The debug logs do show the correct tokens being set 
![image](https://user-images.githubusercontent.com/8843386/79906621-6d890800-8418-11ea-944f-3df44be19bcb.png)

**Comment by dargmuesli** [comment]
Fixed. It's an error that shouldn't be thrown. I'll create a new release in the next few days, when #201 can be closed as well :)

**Comment by dargmuesli** [comment]
#201 seems to become a new major release, thus I published this as `5.0.4`.
